### PR TITLE
userprofile initial commit

### PR DESCRIPTION
**Pull request by angeloremmilbautista**
Developed user profile API endpoint
-added contact number
-added email into signup controller
-resolved code review comments

https://trello.com/c/yXs93DHK

### CODE REVIEW

File: controller/userProfileController.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+let getUser = require('../model/getUserProfile.js')
+
+const updateUserProfile = async (req, res) => {
+  try {
+    const { username, first_name, last_name, email, contact_number } = req.body;
+    if (!username) {
+      return res.status(400).send('Username is required');
+    }
+
+    const user_profile = await updateUser(username, first_name, last_name, email, contact_number)
```
Comment (Eric-Studley) [inline controller/userProfileController.js:11]: This is too many variables to pass down to a function. You already have all of these properties in the req.body you could just pass it down as an object. (or ideally make a copy of the body and pass that down instead).

File: model/updateUserProfile.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const supabase = require('../dbConnection.js');
+
+async function updateUser(username, first_name, last_name, email, contact_number) {
+  console.log(first_name, last_name, email, contact_number);
```
Comment (Eric-Studley) [inline model/updateUserProfile.js:4]: Remove the console log please

**Review by Eric-Studley — CHANGES_REQUESTED** [review]
Looks good overall just a few small tweaks good job

File: controller/userProfileController.js (hunk 1 above)
Comment (angeloremmilbautista) [reply controller/userProfileController.js]: resolved

File: model/updateUserProfile.js (hunk 2 above)
Comment (angeloremmilbautista) [reply model/updateUserProfile.js]: resolved

**Review by Eric-Studley — APPROVED** [review]

**Pull request merged**